### DISCUSSION TITLE
can i return view cshtml file and embed the grid div in it in the new serenity

### DISCUSSION THREAD

**Discussion by heleopless50** [serenity-is/Serenity#7026] [Help]
in older version of serenity 
i was returning cshtml view and through script i define the grid div and specify where the grid should be shown like this ,,, 

<div id="GridDiv"></div>

<script type="text/javascript">
    jQuery(function () {
        new TamkenSoft.Administration.CountriesGrid($('#GridDiv'), {}).init();

        Q.initFullHeightGridPage($('#GridDiv'));
    });
</script>

how can i achieve this in new version of serenity 
thanks in advance

**Comment by VictorTomaili** [comment]
check `AccountPage.cs` and `LoginPage.cshtml` files as a sample.

**Comment by heleopless50** [comment]
i tried to simulate the same process but it give me error 
i did these steps
1- adding the mvc view link
![image](https://github.com/serenity-is/Serenity/assets/87631850/f8f51b2c-6fc9-4168-bb93-7e416bb5b253)

2- in the StudentsPage.cs
![image](https://github.com/serenity-is/Serenity/assets/87631850/91375e05-0378-46f2-94e1-d22814bbd500)

3- adding StudentsPage.cshtml 
![image](https://github.com/serenity-is/Serenity/assets/87631850/0feeb1b2-1af5-40bb-b325-318b49d9780a)

4- and i tried this in StudentPage.tsx
![image](https://github.com/serenity-is/Serenity/assets/87631850/d90abb29-89ea-4488-b8d3-9f320f1ef4fe)

but the console.log() does not execute and i got this error message
![image](https://github.com/serenity-is/Serenity/assets/87631850/6991baa7-c1fa-4171-b724-564e9dbdb806)

what is the problem ?
and should i study react js to understand what is going on ???

**Comment by heleopless50** [comment]
؟؟؟
